cmake/linux/default_libs.cmake

```diff
@@ -47,6 +43,8 @@ if (NOT OS_ANDROID)
     add_subdirectory(base/harmful)
 endif ()
 
+link_libraries(global-group)
+
```

Comment:
I don't understand how this affects how many times `libglibc-compatibility.a` or `libmemcpy.a` will be listed.

Furthermore, I think the `global-group` is linked before `base/harmful` and `base/glibc-compatibility` to give those libraries the same flags as for the rest of the project. Have you thought about this?

Reply:
> I don't understand how this affects how many times libglibc-compatibility.a or libmemcpy.a will be listed.

Because `link_libraries` affects targets defined afterwards, while `glibc-compatibility` itself gets linked into `global-libs`.

>  to give those libraries the same flags as for the rest of the project.

I don't think those flags should apply to libraries listed in `global-libs`. We don't do that for libcxx, libcxxabi, libunwind for instance.